include/iocore/eventsystem/EThread.h

```diff
@@ -479,7 +481,7 @@ class EThread : public Thread
     /// The slices.
     std::array<Slice, N_SLICES> _slice;
 
-    Slice *volatile current_slice = nullptr; ///< The current slice.
```

Comment:
Just curious about what race `current_slice` has. It looks like it's just pointing to one of the `_slice` array and only access from this EThread.

Reply:
It is also accessed from Metrics::record_loop_time() and from Metrics::summarize().  I had to make a new PR because I accidentally pushed my branch to the main repo, not my fork.

Reply:
Ah, `EThread::Metrics::summarize` is called from `EventMetricStatSync`. Got it, thanks.